File: api/code-samples/python/klaytnDebug/runtimeTracing/startGoTrace.py
import json
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

file = "go.trace"

sdk = OpenSDK(host)
debug_response = sdk.debug.start_go_trace(file)

print(json.loads(debug_response.response.data))

File: api/code-samples/javascript/klaytnDebug/runtimeTracing/startGoTrace.js
const OpenSdk = require("opensdk-javascript");

(() => {
    const sdk = new OpenSdk(new OpenSdk.ApiClient("https://api.baobab.klaytn.net:8651"));
    
    const file = "go.trace";

    sdk.debug.startGoTrace(file, {}, (err, data, response) => {
        console.log(data);
    });
}
)()
